apps/coscrad-frontend/src/components/commands/command-selection-area.tsx

```diff
@@ -12,7 +12,11 @@ export const CommandSelectionArea = ({
 }: CommandSelectionAreaProps): JSX.Element => (
     <div data-testid="command-selection-area">
         {metaForCommands.map((commandMeta) => (
-            <CommandButton commandMeta={commandMeta} onButtonClick={onCommandSelection} />
+            <CommandButton
+                key={commandMeta.type}
```

Comment:
This shouldn't appear in the audio annotation page any more, since I've disabled the command panel. But if this is an issue elsewhere, please report it in `QA` and we'll deal with it on a separate branch.